wdl-lint/tests/lints/redundant_input_assignment/source.wdl

```diff
@@ -0,0 +1,28 @@
+#@ except: DescriptionMissing, DisallowedInputName, MissingRequirements
+#@ except: RuntimeSectionKeys
+
+version 1.2
+
+#@ except: SectionOrdering
+task bar {
+    meta {}
+
+    parameter_meta {
+        a: ""
+        b: 2
+        c: ""
+    }
+
+    input {
+        int b
+        String a = a
+        String c = c
+    }
```

Comment:
You are testing the wrong `input `section here. Understandable confusion if you aren't familiar with WDL.
```suggestion
    input {
        int b
        String a
        String c
    }
```
What we're looking at here is the input section to a task (could also be in a workflow). What you're trying to test is the input block of _a workflow call_.
So add something like
```
workflow test {
    input {
        Int b
        String a
        String c
    }
    call bar { input:
         a,  # should not flag
         b = b + 3,  # should not flag
         c = c,  # should flag
   }
}